### ISSUE TITLE
Playstore version 

### ISSUE THREAD

**Issue by bobbymc1985** [SachinVin/citra_android#176]
Is there any difference in the playstore version? I see it was updated sooner rather than on here

**Comment by HelloOO7** [comment]
No. It's just the release build from the AppVeyor CI. There were no updates for months, so it's all the same commit. That's all I know, the Google Play version is in no way associated with this project and it was put up by EnergyCube for better accessibility (similarly to Limbo).

**Comment by bobbymc1985** [comment]
Aw ok. Was just curious as the latest update was on january 29th
